showcase/tests/integration/components/hds/table/index-test.js

```diff
@@ -606,7 +606,30 @@ module('Integration | Component | hds/table/index', function (hooks) {
 
   // aria-labels
 
-  test('it renders the expected `aria-label` values for "select all" and rows (based on provided suffix)', async function (assert) {
+  test('it renders the expected `aria-label` values for "select all" and rows by default', async function (assert) {
+    setSelectableTableData(this);
+    await render(hbs`
+      <Hds::Table
+        @isSelectable={{true}}
+        @model={{this.model}}
+        @columns={{this.columns}}
+        id="data-test-selectable-table"
+      >
+        <:body as |B|>
+          <B.Tr @selectionKey={{B.data.id}}>
+            <B.Td>{{B.data.artist}}</B.Td>
+            <B.Td>{{B.data.album}}</B.Td>
+            <B.Td>{{B.data.year}}</B.Td>
+          </B.Tr>
+        </:body>
+      </Hds::Table>
+    `);
+
+    assert.dom(selectAllCheckboxSelector).hasAria('label', 'Select all rows');
```

Comment:
[question] should be test that the label doesn't change when selected/unselected?

Reply:
Good point! Added explicit checks for both tests.